ckan/templates-bs3/user/edit_user_form.html

```diff
@@ -69,16 +71,16 @@
     {% endblock %}
     {% endif %}
 
+    {% block form_actions %}
     <div class="form-actions">
-      {% block form_actions %}
```

Comment:
Also don't change the scope of existing blocks for the same reason, this particular one seems fine as is

Reply:
Done